mysql-test/suite/rocksdb/r/deadlock_tracking.result

```diff
@@ -59,22 +59,15 @@ KEY
 LOCK TYPE: EXCLUSIVE
 INDEX NAME: PRIMARY
 TABLE NAME: test.t
+ROLLED BACK: NO
 ---------------WAITING FOR---------------
 TXN_ID
 COLUMN FAMILY NAME: default
 KEY
 LOCK TYPE: EXCLUSIVE
 INDEX NAME: PRIMARY
 TABLE NAME: test.t
----------------WAITING FOR---------------
-TXN_ID
-COLUMN FAMILY NAME: default
-KEY
-LOCK TYPE: EXCLUSIVE
-INDEX NAME: PRIMARY
-TABLE NAME: test.t
-
---------TXN_ID GOT DEADLOCK---------
```

Comment:
Actually, what I was going for was that the SHOW ENGINES code can keep using `TRANSACTION ID GOT DEADLOCK`, since I think that's a more concise way of expressing that info.

I think the "rolled back" column makes more sense in the I_S table, than in the show engines command.

Reply:
Ahh, OK, no problem, I can bump that back, but what about the original also re-printing the deadlocked txn info ? Maybe just list the transaction path and finish with the TRANSACTION ID <n> GOT DEADLOCK ?

Reply:
Yeah, we can take out the reprinted transaction.